src/server/controller/cycleData/getBulkDownload/index.ts

```diff
@@ -68,7 +69,7 @@ const handleResult = ({ regions, iso3, name, year, ...row }: Record<string, stri
 
   Object.keys(row).forEach((key) => {
     if (row[key]) {
-      fixed[key] = row[key].replace(/"/g, '').replace(/\n/g, '').replace(/\r/g, '')
+      fixed[key] = `"${row[key].replace(/"/g, '').replace(/\n/g, '').replace(/\r/g, '')}"`
```

Comment:
Wrap values in quotation marks to avoid parsing issues (mainly in Excel, non numeric formats)